docs/api/Model.md

```diff
@@ -1,26 +1,21 @@
 # Model
 
-The Model object represents one DynamoDB table. It takes in both a name and a schema and has methods to retrieve, and save items in the database.
+The Model object represents a table in DynamoDB. It takes in both a name and a schema and has methods to retrieve, and save items in the database.
 
 ## new dynamoose.Model(name, schema[, config])
 
-This method is the basic entry point for creating a model in Dynamoose. When you call this method a new model is created, and it returns a Document initializer that you can use to create instances of the given model.
```

Comment:
Would like to hear more about the reasoning for removing `Document` from the language here. I think it's important to have a reference that it's a Document initializer. That way it's clear that functions like `document.save` work correctly. It doesn't just create an initializer, it creates a Document initializer.

Reply:
I think it was when I started naming document to item. This can be reverted.

Reply:
Reverted